internal/rotate_est.go

```diff
@@ -189,7 +193,10 @@ func decodeEstResponse(estResponse string) (*x509.Certificate, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Invalid pkcs7 data in EST response: %w", err)
 	}
-	return p7.Certificates[0], nil
+	if len(p7.Certificates) < 1 {
+		return nil, errors.New("Invalid pkcs7 data in EST response: no certificates")
```

Comment:
Why do you decided to use `errors.New` instead of fmt.Errorf?

Reply:
There are no format args here... so no need for `fmt.Errorf`.
FYI. Even if I was lazy and used `fmt.Errorf` here cause of a habit, the lint check would fail on it.